experiments/AMIP/user_io/leaderboard/compare_with_obs.jl

```diff
@@ -0,0 +1,19 @@
+##########
+# Precip #
+##########
+
+precip_obs = ObsDataSource(;
+    path = "OBS/gpcp.precip.mon.mean.197901-202305.nc",
+    var_name = "precip",
+    # 1 mm/day -> 1 m/s
+    unit_conversion_factor = Float32(1e-3 / 86400),
```

Comment:
```suggestion
    unit_conversion_factor = Float32(-1/ 86400),
```
(If we have access to parameters here, it's better to use Float32(-1e-3 * ρ_liq / 86400 ) ), where ρ_liq is defined [here](https://github.com/CliMA/ClimaCoupler.jl/blob/b57d37457fb0a12e577f7633596cabea8473dbe2/experiments/AMIP/components/land/bucket_utils.jl#L56)

Reply:
I don't care too much about it. So don't worry about it if it's complicated.